gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-oidc/src/main/java/io/gravitee/am/gateway/handler/oauth2/service/scope/impl/ScopeManagerImpl.java

```diff
@@ -75,7 +79,7 @@ public void afterPropertiesSet() {
 
         logger.info("Register event listener for scopes events for domain {}", domain.getName());
         eventManager.subscribeForEvents(this, ScopeEvent.class, domain.getId());
-
+        this.alwaysProvideEnhancedScopes = OIDC_ALWAYS_ENHANCE_SCOPE.from(environment);
     }
```

Comment:
**question** What is the reason behind replacing @Value with Environment? Here, I believe we're adding another dependency?

Reply:
My point was to centralized the "legacy" setings to easily found them. Environment and @Value are both Spring element so not really new dep (we are still dependant to spring)